modules/common/src/main/scala/notebook/front/widgets/package.scala

```diff
@@ -121,16 +121,36 @@ package object widgets {
             |['observable', 'knockout'],
             |function (O, ko) {
             |  ko.applyBindings({
-            |      value: O.makeObservable(valueId)
+            |      value: O.makeObservable(valueId, initialValue)
             |    },
             |    this
             |  );
             |});
         """.stripMargin,
-        Json.obj("valueId" -> dataConnection.id)
+        Json.obj("valueId" -> dataConnection.id,
+                 "initialValue" -> initialValue)
       )}</p>
   }
 
+  def out = new SingleConnectedWidget[String] {
+    implicit val codec:Codec[JsValue, String] = formatToCodec(None)(Format.of[String])
+
+    lazy val toHtml = <p data-bind="text: value">{
+      scopedScript(
+        """
+          |req(
+          |['observable', 'knockout'],
+          |function (O, ko) {
+          |  ko.applyBindings({
+          |      value: O.makeObservable(valueId)
+          |    },
+          |    this
+          |  );
+          |});
+        """.stripMargin,
+        Json.obj("valueId" -> dataConnection.id)
+      )}</p>
+  }
```

Comment:
`def out = outWithInitialValue(initialValue = "")` do not seem to update the value, if it's set very soon in `out("newValue")`, so just leave to versions on fn. could refactor later.